README.md

```diff
@@ -436,21 +457,46 @@ map<json> ordersMap;
 ``` 
 
-- Here we have used `@kubernetes:Deployment` to specify the docker image name which will be created as part of building this service. 
-- We have also specified `@kubernetes:Service` so that it will create a Kubernetes service which will expose the Ballerina service that is running on a Pod.  
-- In addition we have used `@kubernetes:Ingress` which is the external interface to access your service (with path `/` and host name `ballerina.guides.io`)
+- `@kubernetes:Deployment` is used to specify the Docker image name that should be created as part of building the service.
+- `@kubernetes:Service` is specified to create a Kubernetes service that exposes the Ballerina service that is running on a Pod.
+- `@kubernetes:Ingress` is used as the external interface to access your service (with path `/` and host name `ballerina.guides.io`).
 
-- Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. This will also create the corresponding docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
-  
+Now, use the following command to build a Ballerina executable archive (.balx) of the service that you developed above.
+This creates the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured.
 ```
    $ ballerina build restful_service
-  
-   Run following command to deploy kubernetes artifacts:  
-   kubectl apply -f ./target/restful_service/kubernetes
+   Compiling source
+       restful_service:0.0.0
+
+   Compiling tests
+       restful_service:0.0.0
+
+   Running tests
+       restful_service:0.0.0
+   ballerina: started HTTP/WS endpoint 0.0.0.0:9090
+   ballerina: stopped HTTP/WS endpoint 0.0.0.0:9090
+            ✔ testResourceAddOrder
+            ✔ testResourceUpdateOrder
+            ✔ testResourceFindOrder
+            ✔ testResourceCancelOrder
+
+            4 passing
+            0 failing
+            0 skipped
+
+   Generating executable
+       ./target/restful_service.balx
+            @kubernetes:Service 			 - complete 1/1
+            @kubernetes:Ingress 			 - complete 1/1
+            @kubernetes:Deployment 			 - complete 1/1
+            @kubernetes:Docker 			 - complete 3/3
+
+            Run following command to deploy kubernetes artifacts:
+            kubectl apply -f /home/pubudu/wso2/bbg/restful-service/guide/target/kubernetes/restful_service
 ```
 
-- You can verify that the docker image that we specified in `@kubernetes:Deployment` is created, by using `$ docker images`. 
-- Also the Kubernetes artifacts related our service, will be generated in `./target/restful_service/kubernetes`. 
+- Use the `docker images` command to verify whether the Docker image that you specified in `@kubernetes:Deployment` was created.
+- The Kubernetes artifacts related to the service are generated in the `./target/restful_service/kubernetes` directory.
```

Comment:
Kubernetes artifacts are in ./target/kenernetes/restful_service right?

Reply:
Yes, this should change. Thanks!